examples/frame/src/in_frame/index.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { type InitData, root, useInitData } from "@lynx-js/react";
+import type { GlobalProps } from "@lynx-js/types";
+
+export function App() {
+  let btsInitData: InitData = useInitData();
+  let globalProps: GlobalProps = lynx.__globalProps;
```

Comment:
nitpick: these variables have never changed, use `const` instead.

```suggestion
  const btsInitData: InitData = useInitData();
  const globalProps: GlobalProps = lynx.__globalProps;
```